src/core/map.js

```diff
@@ -1021,6 +1022,63 @@ geo.map = function (arg) {
     return m_this;
   };
 
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * Update the attribution notice displayed on the bottom right corner of
+   * the map.  The content of this notice is managed by individual layers.
+   * This method queries all of the visible layers and joins the individual
+   * attribution notices into a single element.  By default, this method
+   * is called on each of the following events:
+   *
+   *   * geo.event.layerAdd
+   *   * geo.event.layerRemove
+   *
+   * In addition, layers should call this method when their own attribution
+   * notices has changed.  Users, in general, should not need to call this.
+   * @returns {this} Chainable
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this.updateAttribution = function () {
+    // clear any existing attribution content
+    m_this.node().find('.geo-attribution').remove();
+
+    // generate a new attribution node
+    var $a = $('<div/>')
+      .addClass('geo-attribution')
+      .css({
+        position: 'absolute',
+        right: '0px',
+        bottom: '0px',
+        'padding-right': '5px',
+        cursor: 'auto',
+        font: '11px/1.5 "Helvetica Neue", Arial, Helvetica, sans-serif',
+        'z-index': '1001',
+        background: 'rgba(255,255,255,0.7)',
+        clear: 'both',
+        display: 'block',
+        'pointer-events': 'auto'
```

Comment:
It isn't just here, there are a number of places we are injecting style onto the DOM.  I completely agree that it is important to allow the user to override these things.  That is why I added the issue.  Making that change is more work that I feel that I can allocate to it right now.  I think this PR is important to get in now because it is actually a legal issue.